src/scss/jsoneditor/_variables.scss

```diff
@@ -5,7 +5,7 @@ $jse-blue: #3883fa !default;
 $jse-content-color: #1a1a1a !default;
 
 $jse-string: #006000 !default;
-$jse-number: #ee422e !default;
+$jse-number: #6897bb !default;
```

Comment:
What is the reason that you changed the colors for numbers from red to blue? (I would prefer to keep the red color)

Reply:
red just like Error 😂